test/functional/kernel/nested-loop/test-kernel-nested-loop.cpp.in

```diff
@@ -36,12 +36,11 @@ using SequentialKernelNestedLoopExecPols = camp::list<
 using OpenMPKernelNestedLoopExecPols = camp::list<
 
     // Depth 2 Exec Pols
-    NestedLoopData<DEPTH_2, RAJA::seq_exec,  RAJA::omp_for_exec>,
     NestedLoopData<DEPTH_2, RAJA::seq_exec,  RAJA::omp_parallel_for_exec >,
     NestedLoopData<DEPTH_2, RAJA::omp_parallel_for_exec, RAJA::seq_exec >,
     NestedLoopData<DEPTH_2, RAJA::omp_parallel_for_exec, RAJA::simd_exec >,
-    NestedLoopData<DEPTH_2, RAJA::omp_parallel_for_static<8>, RAJA::seq_exec >,
-    NestedLoopData<DEPTH_2, RAJA::omp_parallel_for_static<8>, RAJA::simd_exec >,
+    NestedLoopData<DEPTH_2, RAJA::omp_parallel_for_static_exec<8>, RAJA::seq_exec >,
```

Comment:
All the chunk sizes I'm seeing are quite small.  Is it an attempt to test really small chunk sizes specifically, or deal with a very small loop trip count?

Reply:
This was the way it was before this PR. I just changes the policy syntax. I think the goal here is to test the mechanics rather than usage for performance.